content/en/guide/v10/components.md

```diff
@@ -118,6 +121,7 @@ class Catcher extends Component {
 A `Fragment` allows you to return multiple elements at once. They solve the limitation of JSX where every "block" must have a single root element. You'll often encounter them in combination with lists, tables or with CSS flexbox where any intermediate element would otherwise affect styling.
 
 ```jsx
+// --repl
```

Comment:
Need to define `container` in this one, or just replace it with `document.body` I suppose?

Reply:
Good point, missed that. I kinda want to move folks away from `document.body`. Right now it kinda works because we sort-of make an effort to preserve existing DOM nodes in current Preact, but it always felt a bit risky to me to rely on that behaviour.

In our repl we always add a `<div id="app" />` with this PR to the DOM. Probably best to just switch to that.

Reply:
Oh, didn't know we added that.

I suggested `document.body` as it matches the others on the page. Can leave that for later too, rather than rewriting every sample we may show using it.